app/routes/docs.notices.schema.mdx

```diff
@@ -0,0 +1,210 @@
+---
+handle:
+  breadcrumb: Schema Design
+---
+
+import {
+  Link,
+  ProcessList,
+  ProcessListHeading,
+  ProcessListItem,
+} from '@trussworks/react-uswds'
+
+import { Highlight } from '~/components/Highlight'
+import { Tab, Tabs } from '~/components/tabs/Tabs'
+
+import { feature } from '~/lib/env.server'
+
+export function loader() {
+  return feature('SCHEMA') || new Response(null, { status: 404 })
+}
+
+# GCN Unified Schema
+
+The GCN Unified schema provides a standardized method for organizing GCN notices with common properties across different missions following the uniform unit system. It utilizes JSON (JavaScript Object Notation), a lightweight and easily machine-readable data-interchange format, to define the notice schema. The set of core schemas serves as the foundation for creating new GCN Notices alerts, also allows the inclusion of instrument-specific parameters.
```

Comment:
```suggestion
The GCN Unified Schema provides a standardized method for organizing GCN notices with common properties across different missions following the uniform unit system. It utilizes JSON (JavaScript Object Notation), a lightweight and easily machine-readable data-interchange format, to define the notice schema. The set of core schemas serves as the foundation for creating new GCN Notices alerts, also allows the inclusion of instrument-specific parameters.
```

Also mention JSON Schema.

Reply:
Should re-name as "Unified JSON Schema"?